Traceback (most recent call last):
  File "daemon-example.py", line 135, in <module>
    daemon.start()
  File "/home/ec2-user/Parking-Management-System/panel/daemon.py", line 98, in start
    self.run()
  File "daemon-example.py", line 15, in run
    self.StartDjango()
  File "daemon-example.py", line 29, in StartDjango
    sys.argv[2]="runserver"
IndexError: list assignment index out of range
/home/ec2-user/.virtualenvs/cv/bin/python: can't open file 'daemon-example.py': [Errno 2] No such file or directory
sh: manage.py: command not found
python: can't open file 'manage.py': [Errno 2] No such file or directory
python: can't open file 'manage.py': [Errno 2] No such file or directory
